specification/columns/capacityreservationid.md

```diff
@@ -0,0 +1,37 @@
+# Capacity Reservation ID
+
+A Capacity Reservation ID is the identifier assigned to a [*capacity reservation*](#glossary:capacity-reservation) by the provider. Capacity Reservation ID is commonly used for scenarios to allocate charges for capacity reservation usage.
+
+The CapacityReservationId column adheres to the following requirements:
+
+* CapacityReservationId MUST be present in the billing data when the provider supports *capacity reservations* and MUST be of type String.
+* CapacityReservationId SHOULD NOT be null when a charge is related to a capacity reservation.
```

Comment:
Not blocking on this, but I think we should make this a MUST. While I know it may not be possible by every provider, I would rather see this as a firm statement of what practitioners expect the column to do. We can make the column a SHOULD so it doesn't completely block conformance, but it's important for the column to be defined as practitioners need it to work.
```suggestion
* CapacityReservationId MUST NOT be null when a charge is related to a *capacity reservation*.
```

Reply:
I don't like having a MUST because I don't think we've proven that this is a requirement for FOCUS with ODCRs to work. (It's not how it works in CUR today).